### ISSUE TITLE
RPM Package Name in Unnecessary Lowercase

### ISSUE THREAD

**Issue by Korimsoft** [ctron/rpm-builder#6]
Hello, when we are using your plugin, we would like to have the RPMs case-sensitive. However, current names are created with toLowerCase() in RpmMojo.java, line 711 (makePackageName).

We modified it to
```
private String makePackageName ()
    {
        return this.packageName.trim();
    }
```

and it works like a charm.

**Comment by ctron** [comment]
The recommendation for RPM is to to all lowercase.

But you are right, technically you could use uppercase as well. I would suggest to add a new parameter which allows to disable the enforcing of lowercase. Leaving the default to stick with the current behavior.

**Comment by Korimsoft** [comment]
I know, however our product management wants to have some obscure naming pattern that defies the whole RPM package naming convention (and they are immune to rational arguments). 
We are planning with my colleague to implement a custom pattern parameter if we have some spare time. The default behavior will be as usual but the user will have choice to "mutilate" the package name to their needs.

**Comment by ctron** [comment]
I will try to make a new version this week with a new Package Drone base. Fixing this issue as well.

I am not 100% sure how to do it. As this might break stuff for people which currently depend on this.

But I would still think breaking the old behavior would make sense. So I would suggest to create a new option set `<naming>` which can have sub-elements. One of the (the first) being,  `<case>` which can be set to `lowercase`, `unmodified`, …

If the name contains uppercase characters now and naming option set is missing, then a warning is printed out which tries to explain the breaking change.

**Comment by ctron** [comment]
I pushed a change which should fix this.

Is there a chance you can test this from a SNAPSHOT version of the plugin?

**Comment by Korimsoft** [comment]
Hi, unfortunately, I am leaving the company due to 'political' reasons, so probably I won't be able to test the option soon enough (I would have to set up a dev-env on my personal computer). Anyway, thank you very much for your time. I hope I will be able to make such contributions soon as well.

**Comment by ctron** [comment]
I just released version 0.9.0 which fixes this!
